protocols/1404-mogl/README.md

```diff
@@ -0,0 +1,44 @@
+# Dilution And 1st qPCR
+
+### Author
+[Opentrons](http://www.opentrons.com/)
+
+## Categories
+* Sample Prep
+    * PCR
+
+## Description
+This protocol allows your robot to serially dilute 8 samples in a PCR strip, and use perform PCR on two dilutions in triplicate from each sample. Your PCR strip tubes and output plate will be placed on the TempDecks. Please see Additional Notes for more information on using two TempDecks on the robot.
```

Comment:
Please explain more which temperature deck will be modified in the protocol and which one will not. They need to know what temps to set it to.